components/style/gecko/media_queries.rs

```diff
@@ -35,8 +37,13 @@ pub struct Device {
     pub pres_context: RawGeckoPresContextOwned,
     default_values: Arc<ComputedValues>,
     viewport_override: Option<ViewportConstraints>,
+    /// The font size of the root element
+    root_font_size: AtomicRefCell<Au>,
```

Comment:
This can just look in pres_context.mDocument.mCachedDocumentElement instead, right?